### PR TITLE
TC14 - User can't book more than 10 tickets

### PR DESCRIPTION
**Pull request by sattdn21-01-02**

### CODE REVIEW

File: src/main/java/page_objects/BookTicketPage.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    //Locator
+    private final Dropdown ddlDepartDate = new Dropdown(By.cssSelector("[name=Date]"));
+    private final Dropdown ddlDepartFrom = new Dropdown(By.cssSelector("[name=DepartStation]"));
+    private final Dropdown ddlArriveAt = new Dropdown(By.cssSelector("[name=ArriveStation]"));
+    private final Dropdown ddlSeatType = new Dropdown(By.cssSelector("[name=SeatType]"));
+    private final Dropdown ddlTicketAmount = new Dropdown(By.cssSelector("[name=TicketAmount]"));
+    private final Dropdown btnSubmitBookTicket = new Dropdown(By.cssSelector("input[value='Book ticket']"));
+    private final Label lblErrorMessage = new Label(By.cssSelector(".message.error"));
+    private final Label lblErrorMessageAmount = new Label(By.cssSelector(".validation-error"));
```
Comment (tanmle) [inline src/main/java/page_objects/BookTicketPage.java:21]: again, Vietnamese to English

File: src/test/java/com/logigear/BookTicketTest.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        bookTicketPage.bookTicket(ticket);
+        homePage.goToBookTicketPage();
+        bookTicketPage.bookTicket(ticket);
+
+        String actualErrorMsg = bookTicketPage.getErrorMessage();
+        String actualTicketAmountErrorMsg = bookTicketPage.getErrorAmountTicketMessage();
+        Assert.assertEquals(actualErrorMsg, expectedErrorMsg, "Error message fails to display! ");
+        Assert.assertEquals(actualTicketAmountErrorMsg, expectedTicketAmountErrorMsg, "Ticket amount error fails to display!");
+    }
+
```
Comment (tanmle) [inline src/test/java/com/logigear/BookTicketTest.java:46]: redundant line.

File: src/main/java/page_objects/BookTicketPage.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    private final String dynamicTableCell = "//table/tbody/tr/td[count(//table/tbody/tr/th[.='%s']/preceding-sibling::th)+1]";
```
Comment (tanmle) [inline src/main/java/page_objects/BookTicketPage.java:22]: try to add more property to this xpath, like class or id or something else.

File: src/main/java/page_objects/BookTicketPage.java (hunk 1 above)
Comment (sattdn21-01-02) [reply src/main/java/page_objects/BookTicketPage.java:21]: Sorry i  repeat error again ,  i will overcome it now

File: src/main/java/page_objects/BookTicketPage.java (hunk 3 above)
Comment (sattdn21-01-02) [reply src/main/java/page_objects/BookTicketPage.java:22]: thank you for review, I will fixed mistake now

**Review by tanmle — APPROVED** [review]

**Pull request merged**